src/applications/simple-forms/21-0845/pages/authorizerPersonalInfo.js

```diff
@@ -1,16 +1,24 @@
+import { cloneDeep } from 'lodash';
+
 import fullNameUI from 'platform/forms-system/src/js/definitions/fullName';
 import { schema } from '../../shared/definitions/pdfFullNameNoSuffix';
 
+const authorizerFullNameUI = cloneDeep(fullNameUI);
+
+authorizerFullNameUI.middle['ui:title'] = 'Middle initial';
```

Comment:
Curious about this, because in discussions with Pat we wanted to standardize this pattern, allow the user to type their full name, then only take the first character, rather than explicitly saying to put only one initial. Maybe we could bring this up in standup.

Reply:
Yeah, CAIA changed this back to "...initial".  I'll call this out in Standup, and if he wants it back to "...name" I'll open another PR.